homework/index.js

```diff
@@ -29,22 +29,47 @@
     return elem;
   }
 
+  function createLiElement(ul, element1, element2, options1, options2) {
+    const li = createAndAppend('li', ul, {class: 'list'});
+    return createAndAppend(element1, li, options1),
+      createAndAppend(element2, li, options2);
+  }
+
+  function dateChange(date){
+    return new Date(date).toLocaleString();
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    createLiElement(ul, 'h3', 'a', { text: ' Repository:  ' }, {
+      text: repo.name,
+      href: repo.html_url,
+      target: '_blank',
+    });
+    createLiElement(ul, 'h3', 'span', { text: ' Description:  ' }, { text: repo.description });
+    createLiElement(ul, 'h3', 'span', { text: ' Forks:  ' }, { text: repo.forks });
+    createLiElement(ul, 'h3', 'span', { text: ' Updated:  ' }, { text: dateChange(repo.updated_at) });
   }
 
   function main(url) {
     fetchJSON(url, (err, repos) => {
       const root = document.getElementById('root');
+      createAndAppend('div', root, {
+        text: 'HYF Repositories',
+        class: 'title',
+      });
       if (err) {
         createAndAppend('div', root, {
           text: err.message,
           class: 'alert-error',
         });
         return;
       }
-      const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      repos
+        .sort((a, b) => a.name.localeCompare(b.name))
+        .forEach(repo => {
+          const ul = createAndAppend('ul', root, {class: 'listContainer'});
+          renderRepoDetails(repo, ul);
```

Comment:
You are creating one `ul` for each repo here. You might as well create `ul` directly inside your `renderRepoDetails()` function. Then you do not have to pass it as a function parameter here.